src/Conf.php

```diff
@@ -29,9 +29,9 @@ class Conf
     /**
      * Create new instance of Conf class.
      */
-    public function __construct()
+    public function __construct($filepath)
     {
-        $this->file = storage_path('app/conf.json');
+        $this->file = ($filepath) ? $filepath : storage_path('app/conf.json');
```

Comment:
This wont work.
Actually, better to make config-file with value which contains path to json-file.
It should be publishable. 
In this case path to configuration json would be easy to change anytime. Without interrupting the class constructor.